### PR TITLE
fix:add Pouch_cli_images_test

### PR DESCRIPTION
**Pull request by zeppp**
**1.Describe what this PR did**
Add Pouch_cli_images_test
**2.Does this pull request fix one issue?** 
none
**3.Describe how you did it**
none
**4.Describe how to verify it**
none
**5.Special notes for reviews**
none

### CODE REVIEW

File: test/pouch_cli_images_test.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+func (suite *PouchImagesSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, IsLinux)
+}
+
+// TestPouchImages is to verify pouch images.
+func (suite *PouchImagesSuite) TestPouchImages(c *check.C) {
+	out, err := exec.Command("pouch", "images").Output()
+	c.Assert(err, check.IsNil)
+
+	if !strings.Contains(string(out), "IMAGE") {
```
Comment (allencloud) [inline test/pouch_cli_images_test.go]: I have to say this test may not be so sufficient. I think checking `pouch images` after pulling an image is much better. WDYT? @zeppp

File: test/pouch_cli_images_test.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		"-q":       true,
+		"--digest": true,
+	}
+
+	for arg, ok := range args {
+		cmd := exec.Command("pouch", "images", arg)
+		_, _, err := runCmd(cmd)
+
+		if ok {
+			c.Assert(err, check.IsNil)
```
Comment (allencloud) [inline test/pouch_cli_images_test.go]: Checking err is necessary, but check content in output is also important.

File: test/pouch_cli_images_test.go (hunk 1 above)
Comment (zeppp) [reply test/pouch_cli_images_test.go]: Thanks for your advice!However,how can I know the time once image is pulled successfully?I can not check the err before pulling down.

File: test/pouch_cli_images_test.go (hunk 2 above)
Comment (zeppp) [reply test/pouch_cli_images_test.go]: Thanks for your advice! I will check the content.

**Comment by allencloud** [comment]
https://github.com/alibaba/pouch/pull/130 has been merged and this added more templates there in test. I am afraid conflict happens and a rebase is needed though. Please solve this, and push it back again. @zeppp Thanks a lot.

In addition, please sign off your every commits and squash you commits into a single one. Thanks again.

**Pull request closed without merging**